spec/furniture/journal/entry_component_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe Journal::EntryComponent, type: :component do
+  subject(:output) { render_inline(component) }
+
+  let(:component) { described_class.new(entry: entry) }
+  let(:entry) { create(:journal_entry, body: "https://www.google.com @zee@weirder.earth #GoodTimes") }
+
+  it { is_expected.to have_link("https://www.google.com", href: "https://www.google.com") }
```

Comment:
Any reason you're just testing the link existence and not other strings?

Reply:
Well, I could have done some tests to ensure headings have the appropriate ids and such; but this was the thrust of the work I was doing so it's what I focused on.